### PR TITLE
MediaElch: init at 2.8.2

### PR DESCRIPTION
**Pull request by Stunkymonkey**
###### Motivation for this change
A paket that is missing, which I use a lot. Maybe others are missing it too.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

this is my first submission, so any guidance is wanted.

### CODE REVIEW

**Comment by Stunkymonkey** [comment]
i think the commit hash is not correct. i will fix this shortly

File: pkgs/applications/misc/mediaelch/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  fetchFromGitHub,
+  pkgconfig,
+  qmake,
+  curl,
+  ffmpeg,
+  libmediainfo,
+  libzen,
+  qtbase,
+  qtdeclarative,
+  qtmultimedia}:
```
Comment (gebner) [inline pkgs/applications/misc/mediaelch/default.nix:12]:
```suggestion
  qtmultimedia }:
```

File: pkgs/applications/misc/mediaelch/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  enableParallelBuilding = true;
+
+  prePatch = ''
+    substituteInPlace MediaElch.pro --replace "/usr" "$out"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage    = "https://mediaelch.de/mediaelch/";
+    description = " Media Manager for Kodi";
```
Comment (gebner) [inline pkgs/applications/misc/mediaelch/default.nix:38]:
```suggestion
    description = "Media Manager for Kodi";
```

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -22746,6 +22746,8 @@ in
 
   meh = callPackage ../applications/graphics/meh {};
 
+  mediaelch = libsForQt514.callPackage ../applications/misc/mediaelch { };
```
Comment (gebner) [inline pkgs/top-level/all-packages.nix:22749]:
```suggestion
  mediaelch = libsForQt5.callPackage ../applications/misc/mediaelch { };
```
(Unless it doesn't build with 5.15.)

**Comment by Stunkymonkey** [comment]
hash is now correct and i ran `nixpkgs-fmt`

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -8493,6 +8493,12 @@
     githubId = 638763;
     name = "Stuart Moss";
   };
+  stunkymonkey = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:8496]: Please put this into a separate commit with the message ``maintainers: add stunkymonkey``.

File: pkgs/applications/misc/mediaelch/default.nix [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+  src = fetchFromGitHub {
+    owner = "Komet";
+    repo = "MediaElch";
+    rev = "v${version}";
+    sha256 = "0y26vfgrdym461lzmm5x3z5ai9ky09vlk3cy4sq6hwlj7mzcz0k7";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig qmake ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/mediaelch/default.nix:27]:
```suggestion
  nativeBuildInputs = [ pkg-config qmake ];
```

File: pkgs/applications/misc/mediaelch/default.nix [hunk 6]
```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/mediaelch/default.nix:4]:
```suggestion
, pkg-config
```

File: pkgs/applications/misc/mediaelch/default.nix [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    sha256 = "0y26vfgrdym461lzmm5x3z5ai9ky09vlk3cy4sq6hwlj7mzcz0k7";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig qmake ];
+
+  buildInputs = [ curl libmediainfo libzen ffmpeg qtbase qtdeclarative qtmultimedia ];
+
+  enableParallelBuilding = true;
+
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/mediaelch/default.nix:32]:
```suggestion
```

This is automatically set by qmake.

File: pkgs/applications/misc/mediaelch/default.nix [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  prePatch = ''
+    substituteInPlace MediaElch.pro --replace "/usr" "$out"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://mediaelch.de/mediaelch/";
+    description = " Media Manager for Kodi";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/mediaelch/default.nix:39]:
```suggestion
    description = "Media Manager for Kodi";
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107270` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mediaelch</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107270` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mediaelch</li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 9]
```diff
@@ -22746,6 +22746,8 @@ in
 
   meh = callPackage ../applications/graphics/meh {};
 
+  mediaelch = libsForQt5.callPackage ../applications/misc/mediaelch { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:22749]: that is not sorted alphabetically.

**Review by SuperSandro2000 — APPROVED** [review]
otherwise LGTM
